procedures/release-duty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -7,24 +7,13 @@ Intro
 This manual describes how to set up a new ESR branch. The same process
 can be applied for any branch set up, with slight modifications.
 
-Example tracking bug: `Bug 1644817 - [meta] Support ESR78
-<https://bugzilla.mozilla.org/show_bug.cgi?id=esr78>`__
+Example tracking bug: `Bug 1717527 - tracking bug for build and release
+of Firefox
+91.0esr <https://bugzilla.mozilla.org/show_bug.cgi?id=1717527>`__
 
 Internal changes
 ----------------
 
-This section covers changes to buildbot-configs and tools.
-
-See `Bug 1646588 <https://bugzilla.mozilla.org/show_bug.cgi?id=1646588>`__
-for the details and example changes.
-
-tools changes
-~~~~~~~~~~~~~
-
--  copy the old patcher config, so we can generate partial updates
-   against the previous ESR release. (Not sure if this applies anymore?)
--  add new esr lines to l10n cross-channel `here <https://hg.mozilla.org/integration/autoland/rev/8bcf35827a03386d64e4b89b5b3136bb848b77fe#l1.42>`__ and `here <https://hg.mozilla.org/integration/autoland/rev/8bcf35827a03386d64e4b89b5b3136bb848b77fe#l1.55>`__.
```

Comment:
the cross-channel bit is still relevant, although I've now hooked up bug 1721442 to the esr91 meta bug so maybe we don't need to list it explicitly here.